SSA/Core/Util/ConcreteOrMVar.lean

```diff
@@ -26,6 +26,12 @@ instance : OfNat (ConcreteOrMVar Nat φ) n := ⟨.concrete n⟩
 
 namespace ConcreteOrMVar
 
+/- Terse notation for concrete and mvar. -/
+namespace Notation
+scoped notation "i" n => ConcreteOrMVar.concrete n
+scoped notation "i?" f => ConcreteOrMVar.mvar f
```

Comment:
I'm not totally sold on the value of these notations, though I write them myself. I wanted to mimic LLVM syntax, so it looks like `i 1` and `i? 0` to stand for concrete numbers and mvars, but I'm not a huge fan. @alexkeizer 's second opinion is much appreciated.

Reply:
I dropped the notation for now, given that you are uncertain here.